pom.xml

```diff
@@ -17,7 +17,7 @@
 
     <artifactId>email-ext</artifactId>
     <packaging>hpi</packaging>
-    <version>2.41-SNAPSHOT</version>
+    <version>2.41-ses</version>
```

Comment:
Of course, I just send the pull request as requested in [JENKINS-30854](https://issues.jenkins-ci.org/browse/JENKINS-30854), for easier review.